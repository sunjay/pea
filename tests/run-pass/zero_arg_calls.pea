fn foo() {
    println!(34);
    println!(-921);
}

fn main() {
    println!(1);
    foo();
    foo();
    println!(-630);
    foo();
}
