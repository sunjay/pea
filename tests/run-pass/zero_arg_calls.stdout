1
34
-921
34
-921
-630
34
-921
